package.json

```diff
@@ -25,6 +25,7 @@
     "grunt": "*",
     "grunt-contrib-jshint": "*",
     "grunt-mocha-test": "*",
+    "mocha": "^6.1.4",
```

Comment:
I take it that this was using a global install previously?

Reply:
Presumably, though the travis config doesn't install it in `before_install`.